data_pre_processing.md

```diff
@@ -100,7 +100,7 @@ scp -r <your_user_name>@abel.uio.no:/work/projects/nn9305k/home/<your_user_name>
 ## Trimmomatic - adapter trimming and removing
 
 We wll use [Trimmomatic](http://www.usadellab.org/cms/index.php?page=trimmomatic) to trim/remove adapter and low quality reads.
-This tool is NOT available via _module load_ in abel but available at _/work/projects/nn9305k/bin/_. Make sure you know where the adapter sequences are available.
+This tool is NOT available via _module load_ in abel but available at _`/work/projects/nn9305k/bin/`_. Make sure you know where the adapter sequences are available.
 
```

Comment:
We have gone over to use conda. Could you check if trimmomatic is conda available, and if so, update the text?